### PR TITLE
finished Upgrading logs to TF13 with all test working

### PR DESCRIPTION
**Pull request by mdrummerboy09**
- Test on cloud trail work. Still need aws-config to be updated. 
- Deleting region variables, Since it is not needed for TF13.

### CODE REVIEW

**Review by ralren — COMMENTED** [review]
Can I get an explanation as to why all the region changes? (Not why regions are being removed but why in some places we replace `us-west-*` with something else.)

File: test/terraform_aws_logs_combined_test.go [hunk 1]
```diff
@@ -14,23 +13,27 @@ import (
...
 func TestTerraformAwsLogsCombined(t *testing.T) {
 	// Note: do not run this test in t.Parallel() mode.
+	configName := fmt.Sprintf("aws-config-%s", strings.ToLower(random.UniqueId()))
+	expectedConfigLogsBucket := fmt.Sprintf("terratest-%s", configName)
 
 	tempTestFolder := test_structure.CopyTerraformFolderToTemp(t, "../", "examples/combined")
 	testName := fmt.Sprintf("terratest-aws-logs-%s", strings.ToLower(random.UniqueId()))
 	// AWS only supports one configuration recorder per region.
 	// Each test using aws-config will need to specify a different region.
-	awsRegion := "us-east-2"
```
Comment (mdrummerboy09) [inline test/terraform_aws_logs_combined_test.go:22]: @ralren are you talking about this?

File: README.md [hunk 2]
```diff
@@ -75,8 +71,7 @@ module "aws_logs" {
 ```hcl
 module "aws_logs" {
   source            = "trussworks/logs/aws"
-  s3_bucket_name    = "my-company-aws-logs-lb"
-      region            = "us-west-2"
+  s3_bucket_name    = "my-company-aws-logs-lb"o
```
Comment (dynamike) [inline README.md:74]:
```suggestion
  s3_bucket_name    = "my-company-aws-logs-lb"
```

File: examples/combined/main.tf [hunk 3]
```diff
@@ -2,7 +2,7 @@ module "aws_logs" {
   source = "../../"
 
   s3_bucket_name = var.test_name
-  region         = var.region
+  allow_config   = true
```
Comment (dynamike) [inline examples/combined/main.tf:5]: This shouldn't be needed if default_allow is true

File: main.tf [hunk 4]
```diff
@@ -16,6 +16,11 @@ data "aws_caller_identity" "current" {
 data "aws_partition" "current" {
 }
 
+# testing  to see if Region variable is needed
```
Comment (mdawn) [inline main.tf:19]: this comment needs to come out

File: test/terraform_aws_logs_combined_test.go (hunk 1 above)
Comment (mdawn) [reply test/terraform_aws_logs_combined_test.go:22]: Now that we added the config names as unique, can we change the regions back to the original values and still have passing tests? 

If so we should revert those changes and update the comment above. 🙂

**Review by mdawn — COMMENTED** [review]
PASSING TESTS ARE THE BEST THING ON THE PLANET 🎉

File: test/terraform_aws_logs_combined_test.go (hunk 1 above)
Comment (ralren) [reply test/terraform_aws_logs_combined_test.go:22]: What Misty said.

Comment (mdrummerboy09) [reply test/terraform_aws_logs_combined_test.go]: ahhhhhhhh ok ok. makes sense so I retested it and it works with the original regions and it worked. The regions just had to be different from us-west-2

**Review by dynamike — APPROVED** [review]

**Review by mdawn — APPROVED** [review]

**Comment by mdrummerboy09** [comment]
WOOHOOOOOO!! 🎉  🎉

**Pull request merged**